Style Guide and Diataxis Review - Hornet I (#1315)
gohornet/hornet: documentation/docs/getting_started/bootstrap_from_a_genesis_snapshot.md

---

> description: Please follow these instructions to bootstrap the Chrysalis Phase 2 Hornet node from the Genesis Snapshot.

Reviewer: ```suggestion
description: How to bootstrap the Chrysalis Phase 2 Hornet node from the Genesis Snapshot.
```
